Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,15 +146,25 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
+    /// <summary>
+    /// This event will trigger when the furniture has been changed.
+    /// </summary>
     public event Action<Furniture> Changed;
 
+    /// <summary>
+    /// This event will trigger when the furniture has been removed.
+    /// </summary>
     public event Action<Furniture> Removed;
 
+    /// <summary>
+    /// This event will trigger if <see cref="IsOperating"/> has been changed.
+    /// </summary>
     public event Action<Furniture> IsOperatingChanged;
 
     /// <summary>
-    /// Gets or sets the Furniture's pathfinding modifier which is added into the Tile's final PathfindingCost.
+    /// Gets or sets the Furniture's <see cref="PathfindingModifier"/> which is added into the Tile's final PathfindingCost.
```

Comment:
Agreed, however this is a property and nothing forbid something else then a Tile to use it.
It may be used later on by something else. Tying the comment to the unspecific caller is not a great idea neither. Should we describe what the property means instead of what it is currently used for ?